test/library/standard/Path/expandUser/expandUser.chpl

```diff
@@ -0,0 +1,51 @@
+use FileSystem;
+use Path;
+use Sys;
+
+
+proc test(path: string) {
+  writeln('path= ', path);
+  writeln(expandUser(path));
+}
+
+
+proc doTests(someuser: string) {
+  // A single tilde should expand to home if it is set.
+  test('~');
+
+  // Expand the current user with leading tilde.
+  test('~' + someuser);
```

Comment:
I would also check a path with something other than the user or tildes at the end e.g. `~someuser/foo/bar` (only one such test should be necessary, though, I think this is pretty good coverage)